frontend/src/scenes/persons/PersonScene.tsx

```diff
@@ -312,7 +314,7 @@ export function PersonScene(): JSX.Element | null {
                                           />
                                       </div>
                                       <LemonDivider className="mb-4" />
-                                      <RelatedFeatureFlags distinctId={distinctId || person.distinct_ids[0]} />
+                                      <RelatedFeatureFlags distinctId={distinctId || primaryDistinctId} />
```

Comment:
note that `primaryDistinctId` oftentimes _is_ the first person distinct_id returned from the list, but not always.